src/js/control-bar/mute-toggle.js

```diff
@@ -30,13 +30,6 @@ class MuteToggle extends Button {
     });
   }
 
-  createEl() {
-    return super.createEl('div', {
-      className: this.buildCSSClass(),
-      innerHTML: `<div><span class="vjs-control-text">${this.localize('Mute')}</span></div>`
```

Comment:
I'm not sure if we cleared this up IRL, but `controlText_` is set to "Mute" on the `MuteToggle` class, which is set via `controlText()` in button.